docs/tools/terraform/concepts/data-sources.rst

```diff
@@ -1,5 +1,5 @@
-Data sources in Terraform
-=========================
+Data sources 
```

Comment:
Consider providing more info in this title, e.g.
Data sources for Aiven Terraform Provider or
Data sources for working with Aiven Terraform Provider